waspc/data/Generator/templates/react-app/src/components/_Button.js

```diff
@@ -18,19 +23,39 @@ export class {= button.name =} extends React.Component {
 
   render() {
     return (
+    {=# wasp.isXRayModeEnabled =}
+      <ButtonInfoBox
+        isXRayModeOn={this.props.isXRayModeOn}
```

Comment:
Hm thinking about it now, why do you have to connect this component here to XRay state with isXRayModeOn? Why isn't it already connected, so you would not have to import x ray state here?

Reply:
great idea, done, connected InfoBox directly to Redux.